ansible/roles/service/rds_mysql/files/rds_mysql.yml

```diff
@@ -0,0 +1,135 @@
+---
+
+Description: MySQL RDS setup
+AWSTemplateFormatVersion: '2010-09-09'
+
+Parameters:
+  AllowMajorVersionUpgrade:
+    Description: Switch to allow major version upgrades, should only be enabled temporarily
+    Type: String
+    Default: 'false'
+  BackupRetentionPeriod:
+    Default: 7
+    Description: Automated Backup Retention Period
+    Type: String
+  Environment:
+    Description: Environment name
+    Type: String
+  InstanceType:
+    AllowedValues:
+      - db.m3.medium
+      - db.m3.large
+      - db.m3.xlarge
+      - db.m3.2xlarge
+      - db.r3.large
+      - db.r3.xlarge
+      - db.r3.2xlarge
+      - db.r3.4xlarge
+      - db.r3.8xlarge
+      - db.t2.micro
+      - db.t2.small
+      - db.t2.medium
+      - db.t2.large
+    ConstraintDescription: must be a valid RDS instance type.
+    Default: db.m3.medium
+    Description: Mysql instance type
+    Type: String
+  Port:
+    Description: DB Port number
+    Default: 3306
+    Type: Number
+  Role:
+    Description: Which Role does this instance serve
+    Type: String
+  RootPassword:
+    Description: Mysql Root Password
+    Type: String
+    NoEcho: 'true'
+  Storage:
+    Default: 20
+    Description: Mysql Storage Size
+    Type: Number
+  Version:
+    Description: Engine Version
+    Type: String
+
+Resources:
+  Sg:
+    Type: AWS::EC2::SecurityGroup
+    Properties:
+      GroupDescription: !Sub "MySQL for ${Role}"
+      VpcId:
+        Fn::ImportValue: !Sub "${Environment}-vpc::VpcId"
+      SecurityGroupIngress:
+        - IpProtocol: tcp
+          FromPort: !Ref Port
+          ToPort: !Ref Port
+          CidrIp:
+            Fn::ImportValue: !Sub "${Environment}-vpc::CidrBlock"
```

Comment:
Do we want to instead restrict this to the authoring and site instances, or ideally a security group only they use?

Reply:
yes we could make it more generic and allow to specify SgId

Reply:
yes we could make it more generic and allow to specify SgId